src/controller/java/templates/ClusterInfo-read-interaction.zapt

```diff
@@ -0,0 +1,57 @@
+{{> header}}
+{{#if (chip_has_client_clusters)}}
+
+package chip.devicecontroller;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.List;
+import java.util.HashMap;
+import chip.clusterinfo.ClusterInfo;
+import chip.clusterinfo.InteractionInfo;
+import chip.clusterinfo.CommandParameterInfo;
+import chip.clusterinfo.DelegatedClusterCallback;
+import chip.clusterinfo.ClusterCommandCallback;
+import chip.clusterinfo.CommandResponseInfo;
+import chip.devicecontroller.ChipClusters.DefaultClusterCallback;
+import chip.devicecontroller.ClusterInfoMapping.DelegatedIntegerAttributeCallback;
+
+
+
+public class ClusterReadMapping {
+
+  public Map<String, Map<String, InteractionInfo>> getReadAttributeMap() {
+    ClusterInfoMapping clusterInfoMapping = new ClusterInfoMapping();
```

Comment:
It doesn't really make sense that ClusterInfoMapping invokes ClusterReadMapping which then invokes ClusterInfoMapping again. Below, can we not just do
```
() -> new ClusterInfoMapping.Delegated{{asUpperCamelCase name}}AttributeCallback()
```
?